pages/settings/index.js

```diff
@@ -467,34 +472,39 @@ export default function Settings ({ ssrData }) {
             label={<>don't create referral links on copy</>}
             name='noReferralLinks'
           />
-          <div className='form-label'>content</div>
-          <Checkbox
+          <h4>content</h4>
+          <Input
             label={
-              <div className='d-flex align-items-center'>wild west mode
+              <div className='d-flex align-items-center'>filter posts with fewer than this many sats invested
```

Comment:
Mhh, that's a long label. Would "investment filter" be too confusing even though we have the info icon? Or "sats filter" since I think "investment filter" is also too long and not descriptive enough?

If the label is renamed to "sats filter", I would also rename every use of "investmentFilter" to "satsFilter".